console2/public/cfg.js

```diff
@@ -9,5 +9,7 @@ window.concord = {
                 icon: 'github'
             }
         ]
-    }
+    },
+    defaultRepoBranch: 'master'
```

Comment:
why we need this default value if branch is mandatory?

Reply:
[here](https://github.com/walmartlabs/concord/blob/baf49a25360152125cc1fc1cc01e0af882982c5b/console2/src/components/molecules/GitHubLink/index.tsx#L79)  - when `commitId` is not present but `path` exists

Reply:
hmm... commitId or branch is mandatory now. maybe add `branch` as param for GitHubLink?

Reply:
ok removed default value from console config